docs/DeveloperGuide.md

```diff
@@ -483,18 +483,18 @@ As the users of this application would be hotel mangers of small hotels, there w
-| **Exit**               | `exit`                                                                                                                    | Exits and closes the GuestBook application.                                                                                                               |
-| **Find**               | `find KEYWORD [MORE_KEYWORDS]`                                                                                            | `find James Jake` <br> Searches the entire GuestBook for fields matching "James" and "Jake" and returns the matching guests.                              |
-| **Help**               | `help`                                                                                                                    | Shows a popup on how to get help with GuestBook.                                                                                                          |
-| **List**               | `list`                                                                                                                    | Displays a list containing all the guest in GuestBook.                                                                                                    |
-| **Mark Rooms Unclean** | `markroomsunclean`                                                                                                        | Changes the room clean statuses of all guests to "no".                                                                                                    |                                                                                                                                                       |
+| **Clear**              | `clear`                                                                                                                   | Clears all guest entries in GuestBook.                                                                                                                  |                                                                                                                                                      |
+| **Delete**             | `delete INDEX`                                                                                                            | `delete 3`<br> Deletes the third guest in the current list.                                                                                             |
+| **Edit**               | `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [rm/ROOM] [dr/DATE_RANGE] [ng/NUMBER_OF_GUESTS] [rc/IS_ROOM_CLEAN] [rq/REQUEST]` | `edit 2 rc/yes` <br> Edits the second guest in the current list by changing the guest's the room clean status to "yes".                                 |
```

Comment:
Is there an extra "the" before "room clean"? And can we use code block for `yes`?

Reply:
Yes changed!